### PR TITLE
139 complete manual accessibility tests checklist

### PR DESCRIPTION
**Pull request by jeff-horton-ho-sas**
Is this pull request a content or a code change? (Please fill in the relevant section and delete the other)

# Code change
I can confirm:
## Accessibility considerations
- [x] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed

This PR is retroactively performing the accessibility checks on the site and applying identified fixed

### CODE REVIEW

File: docs/principles/proportionate-security.md [hunk 1]
```diff
@@ -6,12 +6,6 @@ date: 2023-07-13
 tags:
 - Security
 - Software design
-related: # remove this section if you do not need related links on your page
```
Comment (edhamiltonHO) [inline docs/principles/proportionate-security.md:9]: Why has this related section been removed? It is still present in `main`

Comment (edhamiltonHO) [reply docs/principles/proportionate-security.md:9]: Was introduced by this merge commit that resolved a conflict, is a regression as far as I can tell

https://github.com/HO-CTO/engineering-guidance-and-standards/pull/276/commits/7ee1c846c1cc27a6370de9dca8714c578eef590f

Comment (jeff-horton-ho-sas) [reply docs/principles/proportionate-security.md:9]: Yeah this was a merge conflict I misinterpreted - will re-add

**Review by edhamiltonHO — APPROVED** [review]
lgtm

**Review by keithkennedyHO — APPROVED** [review]

**Pull request merged**